packages/client/tests/functional/chunking-query/tests.ts

```diff
@@ -0,0 +1,265 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient, Tag } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(
+  ({ provider, providerFlavor }) => {
+    function generatedIds(n: number) {
+      // ["1","2",...,"n"]
+      const ids = Array.from({ length: n }, (_, i) => i + 1)
+      return ids
+    }
+
+    // Note: we're excluding `sqlite` from the success cases until `createMany` is added
+    // to the SQLite driver. See https://github.com/prisma/prisma/issues/10710.
```

Comment:
This should probably be the explanation in the `optOut` at the bottom instead instead of a comment here?

Reply:
Yes, the placement of this comment has historical reasons that no longer apply. I'm moving it below.